auto_submit/lib/configuration/repository_configuration.dart

```diff
@@ -40,7 +40,7 @@ class RepositoryConfiguration {
     supportNoReviewReverts,
     requiredCheckRunsOnRevert,
   })  : allowConfigOverride = allowConfigOverride ?? false,
-        defaultBranch = defaultBranch ?? 'main',
+        defaultBranch = defaultBranch ?? 'default',
```

Comment:
config.repoDefaultBranch()?

Reply:
I am trying to avoid the config dependency.